migrations/versions/22828d43d921_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
Let's do what we discussed at the meeting and not include any migration files at this stage, I'll generate a single one once we merge stuff.